server/bin/test-bin/test_logger_type.py

```diff
@@ -10,6 +10,7 @@
 _NAME_ = "pbench-logger-test"
 cfg_name = os.environ["_PBENCH_SERVER_CONFIG"]
 logdir = os.environ["LOGSDIR"]
+assert os.path.isdir(logdir), f"LOGSDIR={logdir}"
```

Comment:
Maybe `f"LOGSDIR={logdir!r} isn't a directory"`??

Reply:
Sorry missed this comment before I posted a change.